pyscript.core/src/core.js

```diff
@@ -140,16 +139,17 @@ export const hooks = {
 };
 
 const workerPyScriptModule = [
-    "from pyodide_js import FS",
-    `FS.writeFile('./pyscript.py', ${JSON.stringify(
+    "from pyodide_js import FS as _FS",
+    `_FS.writeFile('./pyscript.py', ${JSON.stringify(
```

Comment:
Why not use pathlib instead?

Reply:
[this is why](https://stackoverflow.com/questions/36015605/prevent-python-packages-from-re-exporting-imported-names) ... I know nothing about Python :sweat_smile: if you have any better way to do this, please share as I have zero opinions here, just trying to avoid leaking `FS` there

Reply:
also please note I `del _FS` after the write so whatever works work for me, but I can confirm `FS` doesn't leak anymore for what I could test.

Reply:
I think doing:
```py
from pathlib import Path
Path("./pyscript.py").write_text(whatever)
del Path
```
would be more idiomatic Python. But this works fine too.

Reply:
I see ... you were focused on the writing, here is the issue is the `FS` leak ... I can use `Path` though, if there is any valid reason to do so, but then again, the `writeFile` is used here and there and I wonder if I can trust it does what it's supposed to do, as that's an utility offered by various interpreters (wasmoon, pyodide, micropython, can't remember if Ruby too)

Reply:
Yeah they mean the same thing. OTOH if you are concerned about making the code interpreter neutral it would probably be better to write this code in JavaScript. Generally I prefer `writeFile` in JavaScript and `pathlib` in Python. But this is also just style guide stuff, certainly not critical either way.

Reply:
... aaaaand it's done! Resolving this, as all the hints/suggestions seem to be in place.